README_tmpl.md

```diff
@@ -0,0 +1,436 @@
+
+# The Leipzig Glossing Rules: Conventions for interlinear morpheme-by-morpheme glosses
+
+Leipzig, last change: May 31, 2015
+
+Further updates will be managed by the Committee of Editors of Linguistics Journals.
+
+
+## About the rules
+
+The Leipzig Glossing Rules have been developed jointly by the Department of
+Linguistics of the Max Planck Institute for Evolutionary Anthropology (Bernard
+Comrie, Martin Haspelmath) and by the Department of Linguistics of the University
+of Leipzig (Balthasar Bickel). They consist of ten rules for the "syntax" and
+"semantics" of interlinear glosses, and an appendix with a proposed "lexicon" of
+abbreviated category labels. The rules cover a large part of linguists' needs in
+glossing texts, but most authors will feel the need to add (or modify) certain
+conventions (especially category labels). Still, it will be useful to have a standard set
+of conventions that linguists can refer to, and the Leipzig Rules are proposed as
+such to the community of linguists. The Rules are intended to reflect common
+usage, and only very few (mostly optional) innovations are proposed.
+We intend to update the Leipzig Glossing Rules occasionally, so feedback is highly
+welcome.
+Important references:
+
+- [Lehmann 1982](cldf/sources.bib#cldf:Lehmann1982)
+- [Croft 2003](cldf/sources.bib#cldf:Croft2003)
```

Comment:
@fmatter This is how I'd mark object references in templates. The nice thing about this is that even without rendering, the templates are somewhat readable. CLDF objects are detected via filename in the URL path and `cldf:` prefix of the fragment. One could also add URL parameters as a way to pass additional information to the template.